.github/workflows/pre-commit.yml

```diff
@@ -11,7 +11,4 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v3
-      - run: pip install pre-commit
-      - run: pre-commit --version
-      - run: pre-commit install
-      - run: pre-commit run --all-files
+      - uses: pre-commit/action@v2.0.3
```

Comment:
Let's not adopt a deprecated Action.  https://github.com/pre-commit/action#readme

https://pre-commit.ci is a better way to go but only @jjjake could make that choice.

Reply:
I completely agree with you. But until @jjjake introduce https://pre-commit.ci/ , I think it is better to use pre-commit/action than now.

Reply:
Relying on a deprecated Action is a bad idea.  Please revert that change.

Reply:
rollbacked:
d9ddb41d5bdde7c0572574af7893d28fc2f3ade2